Sources/ForageSDK/Component/ForagePANTextField/ForagePANTextField.swift

```diff
@@ -279,54 +260,13 @@ public class ForagePANTextField: UIView, Identifiable, ForageElement, ForageElem
     override public var intrinsicContentSize: CGSize {
         return CGSize(width: frame.width, height: 83)
     }
-        
+    
     public func clearText() {
         self.enhancedTextField.text = ""
         self.enhancedTextField.actualPAN = ""
     }
 }
 
-// MARK: - UITextFieldDelegate
-
-extension ForagePANTextField : UITextFieldDelegate {
```

Comment:
these _internal_ methods were showing up in the docs. By moving this to a separate file, it reduces bloat and helps us hide the guts of this from the public-facing reference documentation

Reply:
I'm curious how these are internal? Doesn't the developer tap into these methods to check on the state of the element?